libtock/console.c

```diff
@@ -150,3 +151,29 @@ int getnstr_abort(void) {
   syscall_return_t com = command(DRIVER_NUM_CONSOLE, 3, 0, 0);
   return tock_command_return_novalue_to_returncode(com);
 }
+
+#ifdef _FDEV_SETUP_RW
```

Comment:
What does this do?

Reply:
It's used to printing with picolib, I have added a comment